app/routes/_gcn.docs.code-of-conduct.md

```diff
@@ -0,0 +1,32 @@
+---
+handle:
+  breadcrumb: Code of Conduct
+---
+
+# Code of Conduct
+
+GCN Notices and Circulars are rapid communications which are by definition preliminary, and submitted as best-effort for the accuracy of their content. They are citable in publications, but are not peer reviewed.
+
+The sharing of preliminary analysis and results in an open and respectful manner enables the productive exchange of information and ideas. Notices and Circulars contain factual information on observations and analysis of astronomical transients, not opinions, speculation, or disagreements with individuals or groups. The Circulars [style guide](/circulars/styleguide) provides additional criteria and community standards for Circular content.
+
+## Retractions
+
+If an initial automated or manually-generated Notice/Circular is later deemed to be misclassified as an astronomical object, the generating team should promptly submit another Notice/Circular informing the community of the retracted claim.
+
+## Errata
+
+Submitters can request edits to their own archived Circulars to correct inaccurate information, especially when the correction is important for the follow-up community's activities. These may include author lists, subject corrections, typos, and incorrect citations. This capability should not be used to add details of new observations or analysis &mdash; instead users should submit new Circulars with subjects noting "refined analysis" or "additional observations".
+
+## Reproducing GCN Notices or Circulars in External Archives
+
+The GCN team welcomes cross-compatibility between community astronomical resources. If GCN Notices, Circulars, or their data are used in other systems, please link back to the original source materal at https://gcn.nasa.gov or https://gcn.gsfc.nasa.gov.
```

Comment:
I think that we can be a bit more clear that GCN Circulars are scholarly publications and the normal expectations of scientific conduct for citing and reproducing them apply, since this was a point of uncertainty when we spoke with SCiMMA.